main.py

```diff
@@ -1,16 +1,31 @@
 import uvicorn
 from fastapi import FastAPI
+from pydantic import BaseModel
 
+from firebase import db_operate
 from my_types import HelloWorldType
 
 app = FastAPI()
 
 
+class Item(BaseModel):
+    sente: str
+    gote: str
```

Comment:
`Item`だと何か被りそうなのでこんな感じにしといて欲しいです！
```suggestion
class StartRequestType(BaseModel):
    sente: str
    gote: str
```